frontend/src/components/node/Details.tsx

```diff
@@ -194,6 +201,30 @@ export default function NodeDetails() {
               </AuthVisible>
             ),
           },
+          {
+            id: DefaultHeaderAction.NODE_SHELL,
+            action: (
+              <AuthVisible authVerb="create" item={Pod} namespace={'kube-system'}>
+                <AuthVisible item={Pod} namespace={'kube-system'} authVerb="get" subresource="exec">
+                  <ActionButton
+                    description={
+                      isLinux(item)
+                        ? t('Drop Node Shell')
+                        : t(
+                            'Drop Node Shell is not support on this operatingSystem: {{ device }}',
```

Comment:
Let's shorten/fix it to `Node shell is not supported in this OS: {{ nodeOS }}`...

Reply:
This has been addressed. Resolving.